book/publishing/about.md

```diff
@@ -0,0 +1,49 @@
+(pb-about)=
+
+# About EDS book notebooks
+With the great diversity in open data, software and computational resources, environmental data science research and applications have accelerated rapidly. 
+This presents a timely opportunity for advancing the cyberinfrastructure for showcasing these emerging developments. 
+Building upon existing open science communities such as the [Turing Way](https://the-turing-way.netlify.app/) and [Pangeo](https://pangeo.io), EDS book aims at educating and leveraging good scientific software and data management practises among environmental scientists through peer-reviewed findable, accessible, interoperable and reusable (FAIR) computational notebooks. 
+
+Computation notebooks (CNs) allow users to combine interactive code with text and graphical objects. 
```

Comment:
Unless you're going to be using this abbreviation a lot, I'd write it out fully. You're already using quite a bit of abbreviations which may make it more difficult to read for people that are not familiar with them.

Reply:
Fair point. Fixed in[e5901d7](https://github.com/alan-turing-institute/environmental-ds-book/pull/117/commits/e5901d7bd0b4094a4e5971794dab274a46e32e7c).